script.js

```diff
@@ -0,0 +1,14 @@
+// Select elements
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.navlink');
+// const span = document.getElementsByClassName('bar')[2];
+// Add Event Listener
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('show');
+});
```

Comment:
In order to follow the design as closely as possible, kindly blur the background of the as is on the figma design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/215929743-c37cc635-7bf2-4216-9879-ad4bc8523223.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/215929768-ae56c4c1-a00f-4974-b2a6-934a4f2a1207.png)